File: Programmers/JAVA/lv1/lv1_나누어 떨어지는 숫자 배열.java
https://programmers.co.kr/learn/courses/30/lessons/12910

import java.util.*;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        int[] answer = {};
        Arrays.sort(arr);
        
        ArrayList<Integer> array = new ArrayList<Integer>();
        for(int i = 0; i < arr.length; i++){
            if(arr[i]%divisor == 0){
                array.add(arr[i]);
            }
        }
        
        if(array.isEmpty())
            array.add(-1);
        
        answer = new int[array.size()];
        for(int i = 0; i < answer.length; i++){
            answer[i] = array.get(i);
        }
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_나누어 떨어지는 숫자 배열.cs
https://programmers.co.kr/learn/courses/30/lessons/12910?language=csharp

using System;
using System.Collections.Generic;

public class Solution
{
    public int[] solution(int[] arr, int divisor)
    {
        Array.Sort(arr);
        List<int> list = new List<int>();
        for (int i = 0; i < arr.Length; i++)
        {
            if (arr[i] % divisor == 0)
                list.Add(arr[i]);
        }

        int[] answer = { -1 };
        if (list.Count != 0)
            answer = list.ToArray();

        return answer;
    }
}